### PR TITLE
17 card of talent

### LINKED ISSUE #17: Card of talent
- Створити компонент TalentCard
- Створити файл стилів для компонента TalentCard
- Розмістити всередині картки таланту фото таланту, прізвище, ім'я та спеціалізацію
- Створити окремий компонент для списку головних навичок таланту TalentSkillsList
#8

### PR DESCRIPTION
**Pull request by daniilievtukhov**
There is my work of creating the design of talent cards and rendering them on the page based on the data, which I got from backend developers

### CODE REVIEW

File: src/components/TalentsListPage/TalentsList/TalentsList.jsx [hunk 1]
```diff
@@ -1,5 +1,31 @@
 import styles from "./TalentsList.module.scss";
+import { TalentCard } from "./TalentCard/TalentCard";
+import talents from "./shared/data/talents.json";
 
 export function TalentsList() {
-    return <div className={styles["talents-lists"]}></div>;
+    return (
+        <div className={styles["talents-lists"]}>
```
Comment (IhorKopieichykov) [inline src/components/TalentsListPage/TalentsList/TalentsList.jsx:7]: >
```suggestion
        <div className={styles["talents-list"]}>
```

File: src/components/TalentsListPage/TalentsListPage.module.scss [hunk 2]
```diff
@@ -19,3 +19,12 @@
         color: white;
     }
 }
+.grid-container {
+    display: grid;
+    grid-template-columns: 320px 1fr;
+}
+.grid-filter {
```
Comment (IhorKopieichykov) [inline src/components/TalentsListPage/TalentsListPage.module.scss:26]: You better use flex layout here

File: src/components/TalentsListPage/TalentsList/TalentsList.module.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.talents-lists {
```
Comment (IhorKopieichykov) [inline src/components/TalentsListPage/TalentsList/TalentsList.module.scss:1]:
```suggestion
.talents-list {
```

**Review by IhorKopieichykov — APPROVED** [review]
Ok, but you could better

**Review by Ruslanchik01 — APPROVED** [review]

**Pull request merged**